Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -6,6 +6,17 @@ import format from 'pg-format';
 
 // SQL for Selects
 
+/**
+ * Function that returns a SQL query for checking whether references exist for a given list of document ids
+ * @param documentIds the list of documents to check for existence
+ * @returns SQL query to check for existence ids
+ */
+export const existenceIdsForDocument = (existenceId: string): string =>
+  format(`SELECT existence_id FROM meadowlark.existence WHERE document_id = %L`, existenceId);
+
+export const existenceIdsToVerify = (existenceIds: string[]): string =>
+  format(`SELECT document_id, existence_id FROM meadowlark.existence WHERE existence_id IN (%L)`, existenceIds);
```

Comment:
`existenceIdsToVerify` can be deleted, same behavior as `checkForReferencesByDocumentId `